seth-shaw-unlv/controlled_access_terms
# Github Actions

**GitHub Issue**: (https://github.com/Islandora/documentation/issues/1798)


# What does this Pull Request do?
Converts to github actions
updates the badge
removes travis

# What's new?
* new CI runner

# How should this be tested?
* make sure the tests pass (i don't think they will until this PR is merged - but you can see them running over on the branch itself)
passed here: https://github.com/asulibraries/controlled_access_terms/actions/runs/793821301

# Interested parties
@Islandora/8-x-committers
